sorc/global_cycle.fd/land_increments.f90

```diff
@@ -304,7 +322,9 @@ subroutine add_increment_soil(rla,rlo,stc_state,smc_state,slc_state,stc_updated,
            ! normalize increments
            do k = 1, lsoil_incr
              stc_inc(k) = stc_inc(k) / wsum
+             stc_inc_tmp(ij,k) = stc_inc(k)
              slc_inc(k) = slc_inc(k) / wsum
+             slc_inc_tmp(ij,k) = slc_inc(k)
            enddo
```

Comment:
It looks like it would be possible to merge the two update routines into a single routine. The "JEDI" or native model grid option would skip straight to here. 

The advantage of this is that we don't have to repeat the code below in both routines.